test_sets/vehicle_data_fuelRange.txt

```diff
@@ -0,0 +1,12 @@
+./test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/001_GetVehicleData_fuelRange_all_params.lua
```

Comment:
@HSavynetska I would propose to shorten path, e.g. to `./test_scripts/API/VehicleData/Refactor_Fuel_Information/`